src/server/saved_objects/saved_objects_mixin.js

```diff
@@ -0,0 +1,30 @@
+import { SavedObjectsClient } from './client';
+
+import {
+  createCreateRoute,
+  createDeleteRoute,
+  createFindRoute,
+  createReadRoute,
+  createUpdateRoute
+} from './routes';
+
+export function savedObjectsMixin(kbnServer, server) {
+  const prereqs = {
+    getSavedObjectsClient: {
+      assign: 'savedObjectsClient',
+      method(request, reply) {
+        reply(new SavedObjectsClient(
+          server.config().get('kibana.index'),
+          request,
```

Comment:
It's needed for callWithRequest. This while mixin came from here: https://github.com/elastic/kibana/pull/10967/files#diff-1ebbe7846164a2558bba9ae8d00e88f7R16